data/features/m/m.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+m() {
+  if command -v man >/dev/null; then
```

Comment:
Redirection is not adequate. If `command` fails, an error will show up on the screen.